Sources/swift-parsing-benchmark/JSON.swift

```diff
@@ -97,22 +97,15 @@ let jsonSuite = BenchmarkSuite(name: "JSON") { suite in
       let (name, value) = pair
       object[name] = value
     } element: {
-      Skip {
-        Whitespace()
-      }
+      Whitespace()
       stringLiteral
-      Skip {
-        Whitespace()
-      }
+      Whitespace()
       ":".utf8
       Lazy {
         json!
       }
     } separator: {
       ",".utf8
-      Skip {
-        Whitespace()
-      }
```

Comment:
did you mean to remove this whitespace entirely?

Reply:
Yeah it's actually not needed. I have it fixed in downstream branches, too.